nbs/core.ipynb

```diff
@@ -828,8 +828,8 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #39.            raise NotImplementedError('Backend execution does not support the <code>forecast_fitted_values</code> method')
</pre><p>Execution with a distributed backend only supports <code>forecast</code> and <code>cross validation</code> methods.  Try setting the backend parameter to None.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/statsforecast/pull/303/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>